src/main/java/frc/robot/commands/MoveDrivetrain.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class MoveDrivetrain extends CommandBase {
+  /** Creates a new MoveDrivetrain. */
+  Drivetrain drivetrain;
+
+  public MoveDrivetrain(Drivetrain p_drivetrain) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    drivetrain = p_drivetrain;
+    addRequirements(drivetrain);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    // read direction of joystick
+    drivetrain.getMovement();
+    double speed = 1.0;
+    double rotation = 1.0;
```

Comment:
The goal is to have the speed and rotation be controlled by the joystick right? Take a guess as to how you would do that.